.env.example

```
@@ -62,9 +62,14 @@ ADMIN_AUTH_TOKEN_DURATION=1h
 ADMIN_AUTH_HASH_COST=10
 
-PAYMENT_GATEWAY=midtrans
+PAYMENT_GATEWAY=ipaymu
 PAYMENT_SERVER_KEY=
-PAYMENT_ENVIRONMENT=sandbox
+PAYMENT_VA=
+PAYMENT_BASE_URL=https://sandbox.ipaymu.com
+PAYMENT_RETURN_URL=https://app.cashus.app/payment/return
+PAYMENT_NOTIFY_URL=https://api.cashus.app/api/v1/payments/ipaymu/notifications
+PAYMENT_CANCEL_URL=https://app.cashus.app/payment/return?status=cancel
+PAYMENT_NOTIFY_SECRET=
```

dotenv-linter (4.0.0)

[warning] [UnorderedKey] The PAYMENT_NOTIFY_SECRET key should go before the PAYMENT_NOTIFY_URL key

(UnorderedKey)